lib/passkit/factory.rb

```diff
@@ -2,8 +2,11 @@ module Passkit
   class Factory
     class << self
       # generator is an optional ActiveRecord object, the application data for the pass
-      def create_pass(pass_class, generator = nil)
-        pass = Passkit::Pass.create!(klass: pass_class, generator: generator)
+      def create_pass(pass_class, generator: nil, site_id: nil)
```

Comment:
IMO everything site-related is part of Talkable-specific custom logic and should be in Talkable, not in gem.
Changes here should be more universal so that we could open source our changes.

Reply:
yes, [but](https://github.com/talkable/talkable/blob/0597fe43e3a6646b7ee8dbd6b66f43c5ccba84cb/app/services/passkit/install_url_generator.rb#L7)